src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2652,6 +2652,28 @@ private bool CanDisplayInteractiveGuide(object parameter)
             return !this.ShowStartPage;
         }
 
+        private void StartGettingStartedGuide(object parameter)
+        {
+            try
+            {
+                if (ClearHomeWorkspaceInternal())
+                {
+                    OpenOnboardingGuideFile();
+                    MainGuideManager.LaunchTour(GuidesManager.OnboardingGuideName);
+                }
+            }
+            catch (Exception)
+            {
+                //sidebarGrid.Visibility = Visibility.Visible;
```

Comment:
Log the exception to console here?

Reply:
@RobertGlobant20 Do you recall why we had to deal with exception here and set the visibility?

Reply:
Then I think this line should remain in case the Tour crashes

Reply:
@RobertGlobant20 I cant make this line remain since this is moved to view model. I will see what I can do